src/main/resources/avro/references.avdl

```diff
@@ -81,21 +81,12 @@ record ReferenceSet {
   string id;
 
   /**
-  The IDs of the `Reference` objects that are part of this set.
-
-  This field should be left null if the set contains a very large number of
-  `Reference`s. The `Reference`s in the set may also be obtained through a
```

Comment:
On second thought you're right, it's not necessary. Reference Set -< Reference has a natural hierarchy.

Tangential to this PR:
'client' in my comment was referring to someone writing code against the API, in your view I guess this is a client library developer. If a client is going to be lost just looking at the docs that's probably a problem we should fix. I think depending on divergent sets of documentation in language-specific client libraries being better is not a good solution - in my opinion an ideal client library would be very thin and use as much autogenerated code from this schemas repo as possible (including docs).